hail/python/hailtop/config/deploy_config.py

```diff
@@ -90,6 +90,8 @@ def auth_session_cookie_name(self):
     def external_url(self, service, path, base_scheme='http'):
         ns = self.service_ns(service)
         if ns == 'default':
+            if service == 'site' or service == 'www':
```

Comment:
Pylint is gonna suggest `in`.

Any reason to have the alias? I prefer the one name.

Reply:
OK, I use www everywhere, but left the site deployment and service named site. The router nginx config is the one that maps www to the site service.